pkg/cmd/kubectlenv/kubectlenv.go

```diff
@@ -32,18 +32,30 @@ func NewCmdKubectlEnv(f util.Factory, ioStreams util.IOStreams) *cobra.Command {
 See each sub-command's help for details on how to use the generated script.
 
 The generated script points the KUBECONFIG environment variable to the currently targeted shoot, seed or garden cluster.
+To point the KUBECONFIG to the targeted cluster in all shell sessions consider adding the script to your shell's startup configuration.
 `,
 		Aliases: []string{"k-env", "cluster-env"},
 	}
 	o.AddFlags(cmd.PersistentFlags())
 
 	for _, s := range env.ValidShells() {
+		prompt := s.Prompt(runtime.GOOS)
+		evalCommand := s.EvalCommand(fmt.Sprintf("gardenctl %s %s", cmd.Name(), s))
 		cmd.AddCommand(&cobra.Command{
 			Use:   string(s),
 			Short: fmt.Sprintf("Generate a script that points KUBECONFIG to the targeted cluster for %s", s),
-			Long: fmt.Sprintf("Generate a script that points KUBECONFIG to the targeted cluster for %s.\n\n"+
-				"To load the kubectl configuration script in your current shell session:\n%s\n",
-				s, s.Prompt(runtime.GOOS)+s.EvalCommand(fmt.Sprintf("gardenctl %s %s", cmd.Name(), s)),
+			Long: fmt.Sprintf(`Generate a script that points KUBECONFIG to the targeted cluster for %s.
+
+To load the kubectl configuration script in your current shell session:
+%s
+
+To load the kubectl configuration for each shell session add the following line at the end of the %s file:
+
+    %s
+
+You will need to start a new shell for this setup to take effect.
```

Comment:
same here as in the readme, refer to the previous command

Reply:
Fixed via [`2cd216c` (#490)](https://github.com/gardener/gardenctl-v2/pull/490/commits/2cd216cc2020186e7df92e6c3fe413776e947dce)